consul/data_source_consul_acl_token_secret_id_test.go

```diff
@@ -65,7 +65,7 @@ func TestAccDataACLTokenSecretID_PGP(t *testing.T) {
 			{
 				Config: testAccDataACLTokenSecretIDPGPConfig,
 				Check: resource.ComposeTestCheckFunc(
-					resource.TestCheckResourceAttr("data.consul_acl_token_secret_id.read", "pgp_key", "keybase:terraformacctest"),
+					resource.TestCheckResourceAttr("data.consul_acl_token_secret_id.read", "pgp_key", "keybase:remilapeyre"),
```

Comment:
Yes, the `terraformacctest` does not have a primary key anymore on Keybase which broke the tests:

```
$ curl -s "https://keybase.io/_/api/1.0/user/lookup.json?usernames=terraformacctest&fields=public_keys" | jq '.them[].public_keys.primary.kid'
null
$ curl -s "https://keybase.io/_/api/1.0/user/lookup.json?usernames=remilapeyre&fields=public_keys" | jq '.them[].public_keys.primary.kid'     
"0101f3e0c1554493056a5972000b2eeb73d1ff1bcad37b1153c1054dc309447398660a"
```

Encryption in the Terraform state has been removed from the Terraform SDK (https://www.terraform.io/plugin/sdkv2/guides/v2-upgrade-guide#removal-of-helper-encryption-package) and I think that's why the primary key of this test user has been removed.

While encrypting the Terraform state has major drawbacks we chose to do so for `consul_acl_token_secret_id` because we really don't want to have secret ID of Consul tokens in the state (initially we did not want to provide this resource at all but multiple users have been needing it nonetheless) so I chose to keep the possibility of using PGP encryption for this resource despite the official deprecation in the official Terraform SDK.

Since the key of `terraformacctest` has been removed, I now also need a new test user on Keybase to test this datasource.

Reply:
Thanks for the explanation!